security/build.gradle.kts

```diff
@@ -61,33 +62,46 @@ android {
 //endregion
 
 //region Publish
-val pomPackaging: String by project
 val pomGroup: String by project
 val pomArtifactId: String by project
-/*
- * Maven upload configuration that can be used for any maven repo
- */
-tasks {
-  "uploadArchives"(Upload::class) {
-    repositories {
-      withConvention(MavenRepositoryHandlerConvention::class) {
-        mavenDeployer {
-          withGroovyBuilder {
-            MavenPublish.Bintray.repository(
-              MavenPublish.Bintray.url to uri(MavenPublish.mavenRepo(project))
-            ) {
-              MavenPublish.Bintray.authentication(
-                MavenPublish.Bintray.userName to MavenPublish.mavenUsername(project),
-                MavenPublish.Bintray.password to MavenPublish.mavenPassword(project)
-              )
+
+publishing {
+  publications {
+    create<MavenPublication>("TwilioSecurity") {
+      groupId = pomGroup
+      artifactId = pomArtifactId
+      version = securityVersionName
+      artifact("$buildDir/outputs/aar/security-release.aar")
+
+      pom.withXml {
+        asNode().apply {
+          appendNode("name", "Twilio Security library")
+          appendNode("description", "Twilio Security library")
+          appendNode("url", "https://github.com/twilio/twilio-verify-android")
+          appendNode("licenses").apply {
+            appendNode("license").apply {
+              appendNode("name", "Apache License, Version 2.0")
+              appendNode("url", "https://github.com/twilio/twilio-verify-android/blob/main/LICENSE")
```

Comment:
Just double checking that Apache 2.0 is the proper license.

Reply:
Yes! we are open source. Thanks for checking it